checkov/cloudformation/checks/resource/aws/VPCEndpointAcceptanceConfigured.py

```diff
@@ -0,0 +1,17 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.cloudformation.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class VPCEndpointAcceptanceConfigured(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure that VPC Endpoint Service is configured for Manual Acceptance"
+        id = "CKV_AWS_123"
+        supported_resources = ['AWS::EC2::VPCEndpointService']
+        categories = [CheckCategories.ENCRYPTION]
```

Comment:
Shouldn't it be the same as for Terraform?
```suggestion
        categories = [CheckCategories.NETWORKING]
```

Reply:
yah should, sorry missed it, will change now